lib/locations/repository.rb

```diff
@@ -1,7 +1,29 @@
 module Locations
   class Repository
-    def self.all
-      []
+    def initialize(path = Rails.root.join('lib', 'locations', 'locations.json'))
```

Comment:
Pull the default off the Locations module (or some config object in the Locations area) and it'll be easier to specify an alternative path for test data